com.github.rizinorg.Cutter.yml

```diff
@@ -0,0 +1,29 @@
+#flatpak-builder --force-clean --user --install ./build ./com.github.rizinorg.cutter.yml
+app-id: com.github.rizinorg.Cutter
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
If wayland is supported then this should be `--socket=fallback-x11`

Reply:
Thanks for the tip !